trace-callbacks.bat

```diff
@@ -6,11 +6,8 @@ REM Purpose: Trace every N'Siq CppStyle callback for a given input source file.
 REM          the trace, the rule developer can plan how to write a new rule.
 
 if [%1]==[] goto no_file_given
-if exist filefilter.trace-callbacks.txt del filefilter.trace-callbacks.txt
 
-echo ~ TOOL_trace_nsiqcppstyle_callbacks > filefilter.trace-callbacks.txt
-
-python nsiqcppstyle.py -f filefilter.trace-callbacks.txt %1
+python nsiqcppstyle.py --filter-string="~ TOOL_trace_nsiqcppstyle_callbacks" %1
```

Comment:
Instead of having a separate bat and sh for two platforms, we can achieve the same using a python file.
```
$ python nsiqcppstyle_ast.sh <list of files>
```

Reply:
Do you mean `nsiqcppstyle_ast.py`?  And what is `ast`?

Also, I think the script should accept one, and only one, source file, rather than a list of files.  The purpose of the tool is to provide a callback trace of a single piece of code, not multiple pieces.

Reply:
"ast" = Abstract Syntax Tree module.  What do you have in mind with a script name of `nsiqcppstyle_ast"?  Please explain.

Reply:
The latter. ast was a standin. nsiqcppstyle_trace would work fine as well

Reply:
Nice idea - if the user can run the trace.py script, he obviously has Python 3 ;-).  Will attempt to implement.

Reply:
Implemented as trace-callbacks.py, called like this:

```
python trace-callbacks.py "D:\Junk\NsiqCppStyle\issue_34\hello-world.cpp"
python trace-callbacks.py "D:\Junk\NsiqCppStyle\issue_34\hello-world.cpp" "D:\Junk\NsiqCppStyle\issue_34\hello-world-2.cpp"
```

Done.